spec.yaml

```diff
@@ -109,6 +109,51 @@ paths:
             $ref: "definitions.yaml#/definitions/V4GenericResponse"
 
   /v4/clusters/:
+    get:
+      operationId: getClusters
+      tags:
+        - clusters
+      summary: Get clusters
+      description: |
+        This operation allows to fetch a list of clusters.
```

Comment:
My gut feeling is that I would say "this operation fetches", but I don't see anything wrong with this.